troupon/static/scss/partials/_sections.scss

```diff
@@ -397,3 +397,28 @@ footer {
     margin-top: 30px;
   }
 }
+
+// messages
+.strip {
+  padding: 10px 5px 10px;
+}
+.subject {
+  font-size: 22px;
+  padding-bottom: 5px;
+}
+#compose-form {
+  padding-bottom: 8px;
+}
+.message {
+  background: #ffffff;

```

Comment:
Color literals like `#ffffff` should only be used in variable declarations; they should be referred to via variable everywhere else.